chromium_src/chrome/browser/ui/views/profiles/profile_menu_view.h

```diff
@@ -6,8 +6,7 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_PROFILES_PROFILE_MENU_VIEW_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_PROFILES_PROFILE_MENU_VIEW_H_
 
-#define BRAVE_PROFILE_MENU_VIEW_H    \
-  friend class BraveProfileMenuView; \
+#define BRAVE_PROFILE_MENU_VIEW_H friend class BraveProfileMenuView;
```

Comment:
I guess the patch in the commit went away, so now this commit only fixes formatting and the commit messages has become confusing.

Reply:
I got the patch merged upstream, yes.

Reply:
Good point, reverted here: 11adfb42615f9ff193341add8abc357fee3ce67f.